crates/agentgateway/proto/resource.proto

```diff
@@ -351,7 +351,16 @@ message FrontendPolicySpec {
     Fields fields = 2;
   }
 
-  message Tracing {}
+  message Tracing {
+    string service_name = 1;
```

Comment:
```suggestion
    repeated TracingAttribute resources = 3;
```

of these, service.name is one resource. for example https://opentelemetry.io/docs/languages/js/resources/#adding-resources-with-environment-variables